### ISSUE TITLE
Chain reorganization requires manual refresh in AZ

### ISSUE THREAD

**Issue by ethers** [ethereum/cpp-ethereum#1486]
I see these in the logs:
```
*N* [ 03:50:20 PM | p2p ] Should grab blocks? 24746777345 vs 24746365625 ; 3  blocks, ends dc7937e6…
*N* [ 03:50:20 PM | p2p ] Difficulty of hashchain HIGHER. Grabbing 3 blocks [latest now 6399c307… , was fb990327… ]
```
I thought AZ would show me that the chain had a reorg, but I had to click on Refresh in AZ, to actually see that a reorg had taken place.

**Comment by gavofyork** [comment]
can't reproduce - could you give minimal instructions for reproduction from clean installation?

**Comment by ethers** [comment]
i will check if this is still happening.
A related issue requiring manual refresh is I was on develop e700708.  I rebuilt develop 54f20f1.  Ran eth cli for a few mins, and then ran AZ.  AZ showed only 9088 blocks (had over 30000 before).  The UI stayed at 9088, and I didn't click Refresh.  I restarted AZ (I should have first tried clicking Refresh) and AZ showed 14526 blocks.  I believe I didn't need to restart AZ and clicking Refresh would have updated the UI to show 14526 blocks.
